cmd/goreleaser/internal/configure.go

```diff
@@ -141,6 +143,20 @@ var (
 			newContainerImageManifests(d.name, "linux", k8sArchs, containerImageOptions{}),
 		)
 	}).WithDefaultArchives().WithDefaultChecksum().WithDefaultSigns().WithDefaultDockerSigns().WithDefaultSBOMs().Build()
+
+	// ebpf-profiler distro
+	ebpfProfilerDist = newDistributionBuilder(ebpfProfilerDistro).WithConfigFunc(func(d *distribution) {
+		d.buildConfigs = []buildConfig{
+			&fullBuildConfig{targetOS: "linux", targetArch: ebpfProfilerArchs, ppc64Version: []string{"power8"}},
```

Comment:
Do we need `ppc64Version` here?

```suggestion
			&fullBuildConfig{targetOS: "linux", targetArch: ebpfProfilerArchs},
```